### PR TITLE
NIM on GCP VertexAI Python First Release

### PR DESCRIPTION
**Pull request by FortunaZhang**
NIM on VertexAI Deployment First Release, without airgap support

### CODE REVIEW

File: cloud-service-providers/google-cloud/vertexai/python/nim-vertexai.ipynb [hunk 1]
```diff
@@ -0,0 +1,1372 @@
...
+    "# NIM: llama3-8b-instruct\n",
+    "region = \"us-central1\" # please set here\n",
+    "project_id = None # please set here\n",
+    "public_repository = None # please set here\n",
+    "private_repository = account_name\n",
+    "bucket_url = f\"gs://{account_name}\"\n",
+    "\n",
+    "nim_model = \"nim:llama3-8b-instruct-latest\"\n",
+    "# NIM in NGC\n",
+    "ngc_nim_image = \"nvcr.io/nim/meta/llama3-8b-instruct:latest\"\n",
```
Comment (supertetelman) [inline cloud-service-providers/google-cloud/vertexai/python/nim-vertexai.ipynb:286]: Should try not to use latest tag if possible

**Review by supertetelman — APPROVED** [review]
Recommend some minor changes around latest tag and addition of a pre-amble to the README, otherwise looks good.

File: cloud-service-providers/google-cloud/vertexai/python/nim-vertexai.ipynb (hunk 1 above)
Comment (FortunaZhang) [reply cloud-service-providers/google-cloud/vertexai/python/nim-vertexai.ipynb]: Fixed. Thank you.

**Comment by supertetelman** [comment]
Merging through as-is as discussed. We can add improved usability/readability in a future PR with additional steps documenting air-gap support when necessary.

**Pull request merged**